components/ServiceItem.tsx

```diff
@@ -127,50 +133,54 @@ export const ServiceItem = ({
           />
         )}
       </Stack>
-      <Stack spacing="8px">
-        <Text>{service.description}</Text>
-        {service.email && (
-          <LinkBox>
-            <HStack spacing="8px">
-              <EmailIcon />
-              <LinkOverlay href={`mailto:${service.email}`}>
-                {service.email}
-              </LinkOverlay>
-            </HStack>
-          </LinkBox>
+      {service.organizationNames && service.organizationUrls && (
+        <Text>
+          Group:{' '}
+          <Link href={service.organizationUrls[0]} textDecor="underline">
+            {service.organizationNames[0]}
+          </Link>
+        </Text>
```

Comment:
Not sure if underlining the organization link is the best look to go but this is what it looks like:

<img width="1440" alt="Screen Shot 2022-06-06 at 4 43 38 PM" src="https://user-images.githubusercontent.com/54749942/172245787-64d08d86-a401-41d5-a90d-89ff2f7f151f.png">